src/App.jsx

```diff
@@ -14,70 +16,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
+  const reset = () => setSelected('');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selected ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selected} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+          <button
+            onClick={reset}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      )
+        : (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
```

Comment:
no need to rerender the whole title, because it will not be changed, it is better to use conditional rendering inside `h1`